models/topics-models.js

```diff
@@ -0,0 +1,5 @@
+const db = require("../db/connection.js");
+
+module.exports.selectTopics = (() => {
+    return db.query(`SELECT * FROM topics`);
```

Comment:
As it's data manipulation, you could take the data off the rows key in the model rather than the controller 🤔